code/src/@types/Grid/changesDone.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Grid {
+    export type changesDone = {
+        addedLinesJSON: string;
+        editedLinesJSON: string;
+        hasChanges: boolean;
+        hasInvalidLines: boolean;
+        invalidLinesJSON: string;
+        removedLinesJSON: string;
+    }}
```

Comment:
We should think in refactoring this type, to a class or something else - thinking in structures :)

Reply:
done